src/include/daos_mgmt.h

```diff
@@ -323,6 +323,30 @@ daos_pool_remove_replicas(const uuid_t uuid, const char *group,
 			  d_rank_list_t *svc, d_rank_list_t *targets,
 			  d_rank_list_t *failed, daos_event_t *ev);
 
+/* TODO: get rid of max_nsvc - is it really needed? */
+/**
+ * List all pools created in the specified DAOS system.
+ *
+ * \param group	[IN]		Name of DAOS system managing the service.
+ * \param pools	[OUT]		Array of pool mgmt information structures.
+ * 				NULL is permitted in which case only the
+ * 				number of pools and largest number of service
+ 				replica ranks will be returned in \a npools
```

Comment:
(style)  code indent should use tabs where possible
(style)  please, no space before tabs